src/components/editor/feature-action-panel.tsx

```diff
@@ -102,18 +120,8 @@ export function PureFeatureActionPanelFactory() {
   };
 
   FeatureActionPanel.displayName = 'FeatureActionPanel';
-  FeatureActionPanel.propTypes = {
-    className: PropTypes.string,
-    datasets: PropTypes.object.isRequired,
-    position: PropTypes.object.isRequired,
-    layers: PropTypes.arrayOf(PropTypes.object).isRequired,
-    currentFilter: PropTypes.object,
-    onClose: PropTypes.func.isRequired,
-    onDeleteFeature: PropTypes.func.isRequired
-  };
-
   FeatureActionPanel.defaultProps = {
-    position: {}
+    position: {x: 0, y: 0}
```

Comment:
make default position to be null. and return `null` in render if `position` is `null`

Reply:
fixed
also fixed a test for FeatureActionPanel, plz check it out